backend/curriculum_tracking/api_views.py

```diff
@@ -842,7 +843,8 @@ class WorkshopAttendanceViewset(viewsets.ModelViewSet):
     ]
 
 
-class ManagmentActionsViewSet(viewsets.ViewSet):
+class ManagmentActionsViewSet(viewsets.ViewSet, GenericAPIView):
```

Comment:
Where does `GenericAPIView` come from? I don't see it imported anywhere.

Reply:
Forgot to remove this, it is gone now.